streampipes-client-python/setup.py

```diff
@@ -43,7 +43,7 @@
     "interrogate>=1.5.0",
     "isort>=5.10.1",
     "mypy>=0.990",
-    "pandas-stubs>=1.5.1.221024",
+    "pandas-stubs>=1.2.0.62",
```

Comment:
@kulbachcedric Any specific reason? (just out of curiosity)

Reply:
> @kulbachcedric Any specific reason? (just out of curiosity)
For now, It's for python 3.11 ...